src/Grid/DataGrid.tsx

```diff
@@ -105,13 +105,19 @@ class DataGrid extends React.Component <IProps & WithStyles<keyof typeof styleCl
           searchText: tbResponse.SearchText || '',
           totalRecordCount: tbResponse.TotalRecordCount || 0
         });
+      }, (error: any) => {
+        this.props.errorHandler ? this.props.errorHandler(error) : this.throwError(error);
```

Comment:
Doesn't make any sense to have a method to just throw the error... You were trying to use the ternary operator in an invalid way....